src/app/checks/create/page.tsx

```
@@ -84,20 +88,48 @@ export default async function CreateCheckPage() {
+          </MutliStageRenderer>
+
+          <MutliStageRenderer stage={3}>
+            <SettingsSection />
+          </MutliStageRenderer>
+
+          <MutliStageRenderer stage={4}>
+            <OverviewSection />
+          </MutliStageRenderer>
+        </div>
+        <div className='mx-[1.5%] mt-4 flex justify-between'>
+          <MultiStageBackButton variant='outline' children='Back' />
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)